index.html

```diff
@@ -122,6 +122,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ macmatthew card START ________  -->
+        <div class="card">
+          <p class="name">MacMatthew</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">macmatthew</a>
+          </p>
+          <p class="about">Excited to get into open source contributing!</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="http://www.google.com" target="_blank" title="First Resource">Google</a>
```

Comment:
```suggestion
                <a href="https://www.google.com" target="_blank" title="First Resource">Google</a>
```

Awesome - your pull request looks great. The only thing I'd like you to do is change the links to use https instead of http. Once you've done that, I'll merge your pull request and you'll have made your first contribution!